### ISSUE TITLE
Empty EnumSet can't be used after being deserialized

### ISSUE THREAD

**Issue by francisu** [jdereg/json-io#72]
Adding these lines to the end of TestSet.testSet() shows the problem.

``````
  ```
``````

  testSet._enumSet.remove(ManySets.EnumValues.E1);
        testSet._enumSet.remove(ManySets.EnumValues.E2);
        testSet._enumSet.remove(ManySets.EnumValues.E3);
        json = TestUtil.getJsonString(testSet)
        testSet = (ManySets) TestUtil.readJsonObject(json)
        testSet._enumSet.add(ManySets.EnumValues.E1);

```

The problem is the elementType is null when the EnumSet is deserialized, and then it cannot be used to add values. The elementType needs to be retained during the serialization/deserialization process and it's not.
```

**Comment by jdereg** [comment]
Looking into this.

**Comment by laurgarn** [comment]
I think below merge-request fixes this (too)
https://github.com/jdereg/json-io/pull/180

**Comment by jdereg** [comment]
Fixed in 4.14.1
